src/box/xrow_io.h

```diff
@@ -34,6 +34,9 @@
 extern "C" {
 #endif
 
+#include "small/lsregion.h"
+#include "memory.h"
+
 struct ibuf;
```

Comment:
Might be not the best idea to have inclusions inside `extern "C"`. Can lead to all sorts of funny business with the linker.

Reply:
Fixed, thanks!